README.md

```diff
@@ -1,54 +1,65 @@
-# [`<INSERT MODULE NAME>` module](<INSERT LINK TO MODULE REPO>)
+# youtubeStream modular service
 
-This [module](https://docs.viam.com/registry/#modular-resources) implements the [`<INSERT API TRIPLET>` API]<INSERT LINK TO DOCS (if applicable)> in an <INSERT MODEL> model.
-With this model, you can...
+This module implements the [rdk camera API](https://github.com/rdk/camera-api) in a youtubeStream model.
+With this model, you can provide a youtube URL as a configurable attribute to a camera component.
 
 ## Requirements
 
-_Add instructions here for any requirements._
+The module executable is currently only supported on `darwin/arm64`. Make sure your machine is running on one of these architectures to avoid exec format issues.
 
-```bash
+## Build and Run
 
-```
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `rdk:camera:julie:camera:youtubeStream` model from the [`julie:camera:youtubeStream` module](https://app.viam.com/module/rdk/julie:camera:youtube-stream).
```

Comment:
the model is wrong it should be a triplet of the form `namespace:githubrepo:model`. So it the model id
```suggestion
To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `rdk:camera:julie:something:youtubestream` model from the [`julie:youtubeStream` module](https://app.viam.com/module/rdk/julie:camera:youtube-stream).
```

I would also suggest that we name the module something other than camera.

Reply:
Can the module name simply be youtubeStreamer? I don't see any rules in the docs about the module name. Unless it is supposed to be the same as the module_id? In that case the module_id generated was julie:youtubeStream

Reply:
yep! it can, there are no rules for the module name. but `camera` is quite general and non-descriptive, so we think  about these names first and foremost, since changing them is a breaking change and requires migrations, which cost a lot of time and money. This is not an official module, but it's good practice to do this exercise of thinking of. a good name for your module, and the models within the module.

Reply:
I realize the repo name "camera" is not the best name and maybe something like "streamer" or "video" or "video-streamer" would be better. From my research there are definitely some consequences that could arise from doing this. Do you think it is worth spending the time on to try and change the repo name?

Reply:
I am going to change the name of the repo to be `video-stream` after this PR is closed